lib/3scale/backend/storage_async/client.rb

```diff
@@ -35,7 +35,7 @@ def instance(reset = false)
         end
 
         def initialize(opts)
-          @redis_async = initialize_client(opts)
+          @redis_async = AsyncRedis::Client.call(opts)
```

Comment:
```suggestion
          @redis_async = AsyncRedis::Client.connect(opts)
```
idk, `call` doesn't sound right to me but leave it as you prefer

Reply:
No problem 👍